qemu/tests/netperf_stress.py

```diff
@@ -67,6 +68,10 @@ def netperf_test(duration):
     for server in netperf_server:
         s_info = {}
         if server in vms:
+            params["smp"] = min(params.get_numeric("smp"), 32)
```

Comment:
Hi @heywji 

I think it would be better if we can check the server os type before reset smp/maxcpu, because it is a windows only problem, right?
Btw, you need to update the patch title and commit message like that.
netperf_stress: limit this test to smp **<= 32**

Reply:
Thanks leidong, I have modified it. 

Please help to review it again when you have time : )